systemd-swap

```diff
@@ -582,3 +582,5 @@ case "$1" in
     help
   ;;
 esac
+
+exit 0
```

Comment:
This is a hacky way to solve this issue, but I mean if it works it works.

Reply:
This isn't really an issue though. Bash simply returns the value of the last test/utility if there is nothing else specifying the exit code.

Reply:
No no, it works I know, I just think it looks quite ugly.